scripts/nix/nixpkgs-src.json

```diff
@@ -2,5 +2,6 @@
   "owner": "NixOS",
   "repo": "nixpkgs",
   "rev": "4dd9cd3f69e51dcb69d29a88335f06a544b0e508",
-  "sha256": "0spbpy4c4mvylvdaiic7daw8lfhd4w2mbqx0q2ywmxpqj5hxnhqr"
+  "sha256": "0spbpy4c4mvylvdaiic7daw8lfhd4w2mbqx0q2ywmxpqj5hxnhqr",
```

Comment:
We don't need this.

Reply:
sha256unpacked is needed by fetchNixpkgs, but if we are getting rid of that then it makes sense to not add this.